proposals/NNNN-minimum-main-and-secondary-font-size.md

```diff
@@ -1,5 +1,5 @@
 # Minimum Main and Secondary Font Size
-* Proposal: [SDL-NNNN](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/NNNN-minimum-main-and-secondary-font-size.md)
+* Proposal: [SDL-NNNN](NNNN-filename.md)

```

Comment:
@MichaelCrimando this should be your filename.

[SDL-NNNN](NNNN-NNNN-minimum-main-and-secondary-font-size.md)